plugins/processors/t128_rate/t128_rate.go

```diff
@@ -0,0 +1,164 @@
+package rename
+
+import (
+	"fmt"
+	"math"
+	"sort"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/processors"
+)
+
+const sampleConfig = `
+	## If more than this amount of time passes between data points, the
```

Comment:
If this should match the README I think you're also missing the `[[processors.t128_rate]]` at the top

Reply:
It's strange, but it doesn't match the readme. That piece is added internally by telegraf.